utils/config.go

```diff
@@ -43,6 +43,11 @@ type SSOSetting struct {
 	UserApiEndpoint string
 }
 
+type SSOSettings struct {
+	AllowEmailSignUp bool
```

Comment:
I also just realized the config changes are a breaking change.  I wonder if it does make sense to move out (so it's not a breaking change), since all this will change in 0.8 with the config moving into the db.  Would kind of suck for users to do it twice?